CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [develop]
 
+### Added
```

Comment:
We don't add type definition improvements to the changelog because it's not really related to the UI